README.md

```diff
@@ -474,6 +475,10 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 **[⬆️ Back to Top](#table-of-contents)**
 
+## Personalization
+
+  * [croct.com](https://croct.com) — Personalization platform focused on developer experience, data privacy, and performance. Build native personalization experiences with JavaScript, React, and Next.js. Free plan includes unlimited users, API access, AB testing, and more. 
```

Comment:
We dont add 1 entry categories.  Either move more entries into this category or put it in an existing.

For croct specifically I dont see any pricing information, no information about whats in the free offering etc, so I cant really add it.  Please link to details and also specify whats free in the listing.